setup.py

```diff
@@ -91,7 +91,7 @@
     "quantization": ["bitsandbytes"],
     "scikit": ["scikit-learn"],
     "test": ["parameterized", "pytest-cov", "pytest-rerunfailures", "pytest-xdist", "pytest"],
-    "vllm": ["vllm; sys_platform != 'win32'"],  # vllm is not available on Windows
+    "vllm": ["flask", "vllm; sys_platform != 'win32'"],  # vllm is not available on Windows
```

Comment:
Not super important, but I personally prefer [`fastapi`](https://fastapi.tiangolo.com) for HTTP servers since it's both more performant and has lots of nice features like [request validation](https://fastapi.tiangolo.com/tutorial/query-params-str-validations/)

Reply:
Also, should we put a lower bound on `vllm` since there's lots of important fixes in the latest version